pkg/config/config.go

```diff
@@ -16,16 +15,25 @@ var (
 		field.WithDescription("The Tenable API key connect to the Tenable API"),
 		field.WithRequired(true),
 	)
+	EnableOnProvisionField = field.BoolField(
```

Comment:
this, AccessKeyField and SecretKeyField need a WithDisplayName and must match what's in C1 as per 

https://conductorone.atlassian.net/wiki/spaces/ENG/pages/2410315780/Containerizing+Connectors+Guide